src/plugins/condition/components/Criterion.vue

```diff
@@ -118,21 +120,17 @@ export default {
             inputTypes: INPUT_TYPES
         }
     },
-    watch: {
-        telemetry: {
-            handler(newTelemetry, oldTelemetry) {
-                this.checkTelemetry();
-            },
-            deep: true
-        }
-    },
     computed: {
         setRowLabel: function () {
             let operator = this.trigger === 'all' ? 'and ': 'or ';
             return (this.index !== 0 ? operator : '') + 'when';
         },
         filteredOps: function () {
-            return [...this.operations.filter(op => op.appliesTo.indexOf(this.operationFormat) !== -1)];
+            if (this.operationFormat === 'all') {
+                return this.operations;
+            } else {
+                return [...this.operations.filter(op => op.appliesTo.indexOf(this.operationFormat) !== -1)];
```

Comment:
Why are we creating a copy here? Filter already does that for us.

Reply:
Good point. Was existing code. I'll change it.

Reply:
Done.